scipy/sparse/linalg/_special_sparse_arrays.py

```diff
@@ -20,24 +20,22 @@ class LaplacianNd(LinearOperator):
         A tuple of integers of length ``N`` (corresponding to the dimension of
         the Lapacian), where each entry gives the size of that dimension. The
         Laplacian matrix is square of the size ``np.prod(grid_shape)``.
-    boundary_conditions : {'neumann', 'dirichlet', 'periodic'}, optional
+    boundary_conditions : {"neumann", "dirichlet", "periodic"}, optional
         The type of the boundary conditions on the boundaries of the grid.
-        Valid values are ``'dirichlet'`` or ``'neumann'``(default) or
-        ``'periodic'``.
+        Valid values are ``"dirichlet"`` or ``"neumann"``(default) or
+        ``"periodic"``.
     dtype : dtype
         Numerical type of the array. Default is ``np.int8``.
 
-    Attributes
-    ----------
-    eigenvalues : ndarray
-        Eigenvalues of the Laplacian matrix in ascending order.
-
     Methods
     -------
     toarray()
         Construct a dense array from Laplacian data
     tosparse()
         Construct a sparse array from Laplacian data
+    eigenvalues()
```

Comment:
The full call signature should be here. The following suggestion uses my suggested API from a previous comment.

```suggestion
    eigenvalues(m=None, largest=True)
```

Reply:
done, without the "largest" that will be unused in this PR